app/views/users/_user.html.haml

```diff
@@ -0,0 +1,12 @@
+%li#show-user
+  %a{href: user_profile_path(user.nickname), id: 'profile'}
+    - if user.profile_picture.attached?
+      = image_tag (user.profile_picture), alt: 'profile picture', class: 'profile-picture'
+    - else
+      = image_tag 'no_profile_picture.png', alt: 'no profile picture', class: 'profile-picture'
+    .content
+      .p
+        %span.name #{user.first_name} #{user.last_name}
```

Comment:
You are doing something similar in your user profile sidebar. What do you think about a function in the user model that returns the user's full name?

Reply:
rails best practices throw "remove unused methods (User#full_name)" even though I use the method in the views :/